test/exercises/duplicates/forStudents/duplicates.chpl

```diff
@@ -0,0 +1,117 @@
+// This version is perhaps the most obvious / easiest to parallelize
+
+use FileHashing;
+use FileSystem;
+use Sort;
+
+config const filter = "";
+
+// computeHashes is called by the below code to
+// fill in the hash components of the array hashAndPath.
+//
+// hashAndPath is an array of tuples. Each array element
+// has two tuple components:
+//   component 1 is the hash
+//   component 2 is the path
+//
+proc computeHashes(hashAndPath:[] (SHA256Hash, string)) {
+
+  // The Tuples and Arrays primers might be useful
+  // if you're not sure how to write this:
+
+  // https://chapel-lang.org/docs/master/primers/tuples.html
+  // https://chapel-lang.org/docs/master/primers/arrays.html
+
+  // call computeFileHash to gather the hashes
+  // computeFileHash is declared like this:
+  //   computeFileHash(path: string): SHA256Hash throws
+  // so it accepts a string argument and returns a SHA256Hash
+
+
+}
+
+proc handleArguments(args: [] string, ref paths: domain(string)) {
```

Comment:
It might be helpful to print an error message when no arguments are given - better yet, print some help output for the program.

Reply:
e.g.

```
$ ./duplicates
No directory provided.

usage: 
duplicates <directory>
```